spigot/src/main/java/org/dynmap/bukkit/Helper.java

```diff
@@ -40,6 +40,12 @@ else if(Bukkit.getServer().getClass().getName().contains("GlowServer")) {
                 Log.info("Loading Glowstone support");
                 BukkitVersionHelper.helper = loadVersionHelper("org.dynmap.bukkit.helper.BukkitVersionHelperGlowstone");
             }
+            else if (v.contains("(MC: 1.21.7")) {
+                BukkitVersionHelper.helper = loadVersionHelper("org.dynmap.bukkit.helper.v121_7.BukkitVersionHelperSpigot121_7");
+            }
+            else if (v.contains("(MC: 1.21.6")) {
+                BukkitVersionHelper.helper = loadVersionHelper("org.dynmap.bukkit.helper.v121_6.BukkitVersionHelperSpigot121_6");
+            }
```

Comment:
For this PR, you created two separate bukkit helpers, but the api hasn't changed between 1.21.6 and 1.21.7. It's still v1_21_R5 for both versions. I think you should delete one or the other, and use the same helper in both.